pages/common/docker-image-pull.md

```diff
@@ -17,7 +17,7 @@
 
 - Download a Docker images for a specific platform, e.g. linux/amd64:
 
-`docker {{[pull|image pull]}} --platform {{linux/amd64}} {{image}}:{{tag}}`
+`docker {{[pull|image pull]}} --platform linux/amd64 {{image}}:{{tag}}`
```

Comment:
Not sure if this is necessary, as the instructions are vague enough. It's not saying that "this command will do this", but "but this command can many things, for example this"

Reply:
Was doubting myself as well, but @dmmqz requested it; https://github.com/tldr-pages/tldr/pull/20359#discussion_r2651307335

Reply:
Sounds fair. It's duplicated information anyways since the placeholder already contains that info